### ISSUE TITLE
Accessibility Issue - opacity makes contrast too low

### ISSUE THREAD

**Issue by edent** [NationalSecurityAgency/nationalsecurityagency.github.io#37]
[Section 508](https://www.section508.gov/) is a legal requirement to ensure the accessibility of services offered to the public. Web Content Accessibility Guidelines (WCAG 2.0) is a series of checkpoints which [ensures that  websites are accessible](https://www.usa.gov/accessibility).

The "View Repo" button's contrast is too low. This causes the document to fail an automated accessibility audit:

![Screenshot showing a WCAG failure.](https://user-images.githubusercontent.com/837136/193409757-8d44219b-c5b2-42f1-a723-35403b3bb5df.png)

This appears to be caused by `.usa-hero-callout { opacity: 0.9; }`

While the aesthetic effect of seeing the stars behind the text is attractive, it does make the text difficult to read.
